docs/docs/targeting.md

```diff
@@ -0,0 +1,196 @@
+Part of what makes Pathfora powerful is its ability to interact with [Lytics](http://www.getlytics.com/) and target certain audiences with certain modules. Setting up the targeting in Pathfora requires an object with certain targeting rules as the first param to [initializeWidgets](/api/methods.md#initializewidgets) and your Lytics Account ID as a second parameter.
```

Comment:
`Part of what makes Pathfora powerful is its ability to interact with [Lytics](http://www.getlytics.com/) and target certain audiences with certain modules. Setting up the targeting in Pathfora requires an object with certain targeting rules as the first param to [initializeWidgets](/api/methods.md#initializewidgets) and your Lytics Account ID as a second parameter.` --> `Pathfora gains power by seamlessly integrating with [Lytics](http://www.getlytics.com/) for real-time user identification. This allows for precise target audience targeting with each module. Setting up the targeting in Pathfora requires an object with certain targeting rules as the first param to [initializeWidgets](/api/methods.md#initializewidgets) and your Lytics Account ID as a second parameter.`